cmd/minishift/cmd/root.go

```diff
@@ -71,6 +74,15 @@ var RootCmd = &cobra.Command{
 
 		ensureConfigFileExists(constants.ConfigFile)
 
+		// Create MACHINE_NAME.json
+		jsonDataPath := filepath.Join(constants.Minipath, "machines", constants.MachineName+".json")
```

Comment:
@hferentschik Interesting, if I add space around `+`, then `go fmt` complains about it. Weird .